script/seed.js

```diff
@@ -24,23 +24,39 @@ async function seed() {
     Skill.create({title: 'Chef'}),
     Skill.create({title: 'Coordinator'}),
     Skill.create({title: 'Dancer'}),
-    Skill.create({title: 'Decorations'}),
+    Skill.create({title: 'Decorator'}),
     Skill.create({title: 'DJ'}),
-    Skill.create({title: 'Lighting'}),
+    Skill.create({title: 'Lighting Technician'}),
     Skill.create({title: 'Magician'}),
     Skill.create({title: 'MC'}),
     Skill.create({title: 'Server'})
   ])
 
   const events = await Promise.all([
     Event.create({location: 'Chicago'}),
-    Event.create({location: 'Texas'})
+    Event.create({location: 'Chicago'}),
```

Comment:
be sure to use Event.setHost(<user>) because the data flow in creating an event from the Front end sets host by default. So you'll wanna associate a user with an event here